subsys/bluetooth/host/att.c

```diff
@@ -3012,6 +3012,18 @@ static void bt_att_connected(struct bt_l2cap_chan *chan)
 		le_chan->rx.mtu = BT_ATT_DEFAULT_LE_MTU;
 	}
 
+	if (le_chan->psm == BT_EATT_PSM) {
```

Comment:
Could this condition just be an `else` from the `if` above?

Reply:
you're right. Although I'm not sure of the coding style with the ifdef.